config.py

```diff
@@ -49,7 +49,7 @@ def main():
         os.popen('cp ' + dir_path + '/sysconfigs/repos.yaml ' + args.reposdir)
 
     # configure config.yaml
-    config_data = yaml.safe_load(open('sysconfigs/config.yaml', 'r'))
+    config_data = yaml.safe_load(open('sysconfigs/' + args.machine.replace('admin-', '') + '/config.yaml', 'r'))
```

Comment:
The machine args are either `admin-tsa` or `admin-daint`?
In the README.md there is an example with `-m tsa` :thinking: 

Or is the idea that this will work for both `tsa` and `admin-tsa`?

Reply:
Yeah because lambda users are not supposed to know about admin-tsa, second statement is exact !